app/models/submission.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Submission < ApplicationRecord
+  # Associations
+  belongs_to :submitter, class_name: 'User'
+  belongs_to :challenge
+  belongs_to :phase
+  belongs_to :manager, class_name: 'User'
+
+  # Fields
+  attribute :title, :string
+  attribute :brief_description, :string
+  attribute :brief_description_delta, :string
+  attribute :description, :string
+  attribute :description_delta, :string
+  attribute :external_url, :string
+  attribute :status, :string
+  attribute :judging_status, :string, default: "not_selected"
+  attribute :terms_accepted, :boolean, default: nil
+  attribute :review_verified, :boolean, default: nil
+
+  # Virtual Fields
+  attribute :brief_description_length, :integer
+
+  # Validations
+  validates :title, :start_date, :end_date, presence: true
```

Comment:
there are no start_date or end_date fields on this model
```suggestion
  validates :title, presence: true
```